### PR TITLE
Validate email addresses with special characters

### PR DESCRIPTION
**Pull request by AntoineAugusti**
Related to https://github.com/cds-snc/notification-api/issues/760

### CODE REVIEW

File: notifications_utils/__init__.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # Note: Normal apostrophe eg `Firstname-o'surname@domain.com` is allowed.
 hostname_part = re.compile(r'^(xn-|[a-z0-9]+)(-[a-z0-9]+)*$', re.IGNORECASE)
 tld_part = re.compile(r'^([a-z]{2,63}|xn--([a-z0-9]+-)*[a-z0-9]+)$', re.IGNORECASE)
-VALID_LOCAL_CHARS = r"a-zA-Z0-9.!#$%&'*+/=?^_`{|}~\-"
+VALID_LOCAL_CHARS = r"a-zA-ZÀ-ÿ0-9.!#$%&'*+/=?^_`{|}~\-"
```
Comment (AntoineAugusti) [inline notifications_utils/__init__.py:10]: Obvious StackOverflow link https://stackoverflow.com/a/26900132

**Review by brdunfield — APPROVED** [review]
👍

**Comment by AntoineAugusti** [comment]
Created a new tag + release after merging this PR. Can you confirm this is all I was supposed to do?

**Comment by brdunfield** [comment]
Yep! I see the tag on master 👍

**Pull request merged**